docs/typescript.md

```diff
@@ -0,0 +1,14 @@
+# usage of TypeScript
+
+Our use of TypeScript has to accomodate both .js development in agoric-sdk (which cannot import types) and .ts development of consumers of agoric-sdk packages (which can import types). For .js development, we want ambient (global) types so that we don't have to proceed each type reference by an import. For .ts development, we want exports from modules so we don't pollute a global namespace.
```

Comment:
```suggestion
Our use of TypeScript has to accomodate both .js development in agoric-sdk (which cannot import types) and .ts development of consumers of agoric-sdk packages (which can import types). For .js development, we want ambient (global) types so that we don't have to precede each type reference by an import. For .ts development, we want exports from modules so we don't pollute a global namespace.
```